src/awsexplorer/utils.ts

```diff
@@ -0,0 +1,33 @@
+/*!
```

Comment:
Added util file to hold `fileIconPath` and `folderIconPath` functions

Reply:
Seems like these functions would fit in `src/test/shared/utilities/iconPathUtils.ts` (which already exists).

In general we should avoid having lots of over-specific util modules. Even `iconPathUtils.ts` is over-specific, but at least putting these functions in it avoids a new util module.